cypress/support/constants.ts

```diff
@@ -1,6 +1,66 @@
+import { EdaCredential } from '../../frontend/eda/interfaces/EdaCredential';
+import { EdaDecisionEnvironment } from '../../frontend/eda/interfaces/EdaDecisionEnvironment';
+import { EdaProject } from '../../frontend/eda/interfaces/EdaProject';
+import { EdaRulebookActivation } from '../../frontend/eda/interfaces/EdaRulebookActivation';
+import { EdaCredentialType } from '../../frontend/eda/interfaces/EdaCredentialType';
+
 export enum SERVER_NAME {
   AWX_SERVER = 'AWX Ansible Server',
   EDA_SERVER = 'Event Driven Automation Server',
   HUB_SERVER = 'Automation Hub',
   GALAXY_SERVER = 'Galaxy Server',
 }
+
+type ResourceObject =
+  | EdaProject
+  | EdaDecisionEnvironment
+  | EdaRulebookActivation
+  | EdaCredential
+  | EdaCredentialType;
+
+export interface AccessTabResource {
+  name: string;
+  content_type: string;
+  creation: (() => Cypress.Chainable<ResourceObject>) | null;
+  deletion: (resourceObject: ResourceObject) => Cypress.Chainable<void>;
+  role: string;
+}
+export const user_team_access_tab_resources: AccessTabResource[] = [
```

Comment:
love the idea of using an array of objects to get all these tests in, awesome work! @appuk